azure-pipelines.yml

```diff
@@ -110,8 +122,8 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
     - task: PublishBuildArtifacts@1
-      displayName: Publish Release DMG
-      condition: eq(variables['B_BUILD_TYPE'], 'Release')
+      displayName: Publish Release DMG if on Big Sur agent
+      condition: and(eq(variables['B_BUILD_TYPE'], 'Release'), eq(variables['imageName'], 'macOS-11'))
       inputs:
         pathtoPublish: build/bundle
-        artifactName: Mac Release Disk Image and App $(imageName)
+        artifactName: Universal Binary for macOS
```

Comment:
Since we actually disable the arm64, it's not actually universal.

Reply:
This was leftover from a previous commit where arm64 was enabled. Removed in a9bcc38d.